docker/build_docker_sagemaker.sh

```diff
@@ -10,26 +10,12 @@ else
     GSF_HOME="$1"
 fi
 
-# process argument 2: docker image type, default is GPU
-if [ -z "$2" ]; then
-    IMAGE_TYPE="gpu"
-else
-    IMAGE_TYPE="$2"
-fi
-
-# process argument 3: docker image name, default is graphstorm
-if [ -z "$3" ]; then
-    IMAGE_NAME="graphstorm"
-else
-    IMAGE_NAME="$3"
-fi
-
-# process argument 4: image's tag name, default is sm
-if [ -z "$4" ]; then
-    TAG="sm"
-else
-    TAG="$4"
-fi
+# # process argument 2: docker image type, default is GPU
```

Comment:
If you are going to change this script, it is better to create another PR. I don't think you change the functionality of this script.